templates/blog/post_list.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>Blog Posts</title>
+{% endblock %}
+
+{% block content %}
+  <h2>Posts</h2>
+  <div>
+    <ul>
+      {% for post in post_list %}
+        <li>
+          <h3><a href="{% url 'blog:post-detail' pk=post.id %}">{{ post.title }}</a></h3>
+          <div> Author: {{ post.owner }}</div>
+          <div>{{ post.created_time }}</div>
+          <div>{{ post.content }}</div>
```

Comment:
You are displaying the entire content of the post on the list page. This might make the page very long if the posts are long. Consider showing only a preview of the content here, and the full content on the detail page.